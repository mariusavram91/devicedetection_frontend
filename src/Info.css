.Info table {
  font-size: 8px;
  margin-left:auto;
  margin-right:auto;
}
