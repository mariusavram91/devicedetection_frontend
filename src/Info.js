import React, { Component } from 'react';

import './App.css';

class Info extends Component {
  render() {
      return(
        <div className="Info">
            <table>
              <tbody>
                <tr>
                  <th>Type of device</th>
                  <th>Device OS</th>
                </tr>
                <tr>
                    <td>{ this.props.info.type }</td>
                    <td>{ this.props.info.os }</td>
                </tr>
              </tbody>
            </table>
        </div>
      )
  }
}

export default Info;
