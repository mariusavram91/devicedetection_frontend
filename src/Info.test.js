import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Info from './Info';

describe('Info', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Info info={{ type: 'smartphone', os: 'iOS'}} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
