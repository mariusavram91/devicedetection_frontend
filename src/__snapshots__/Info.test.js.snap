// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info renders correctly 1`] = `
<div
  className="Info"
>
  <table>
    <tbody>
      <tr>
        <th>
          Type of device
        </th>
        <th>
          Device OS
        </th>
      </tr>
      <tr>
        <td>
          smartphone
        </td>
        <td>
          iOS
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
